src/marqo/core/distributed_lock/distributed_lock_manager.py

```diff
@@ -0,0 +1,17 @@
+from marqo.vespa.marqo_zookeeper_client import MarqoZookeeperClient
+from marqo.core.distributed_lock.zookeeper_distributed_lock import ZookeeperDistributedLock
+
+_DEPLOYMENT_LOCK_PATH = "/marqo__deployment_lock"
+
+
+def get_deployment_lock(zookeeper_client: MarqoZookeeperClient, acquire_timeout=1) -> ZookeeperDistributedLock:
```

Comment:
I wonder if you could simplify this and add it to `zookeeper_distributed_lock.py`. It makes even more sense when you realize this is just a factory method for the lock class. Really good pattern otherwise